jp/components/texthighlight.md

```diff
@@ -0,0 +1,345 @@
+---
+title: TextHighlight ディレクティブ - ネイティブ Angular | Ignite UI for Angular
+_description: Ignite UI for Angular TextHighlight ディレクティブは、テキスト部分のハイライトやアクティブ ハイライトがあります。
+_keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェット, web ウィジェット, UI ウィジェット, Angular, ネイティブ Angular コンポーネント スイート, ネイティブ Angular コンポーネント, ネイティブ Angular コントロール, ネイティブ Angular コンポーネント ライブラリ, Angular TextHighlight ディレクティブ, IgxTextHighlight ディレクティブ
+_language: ja
+---
+
+## TextHighlight ディレクティブ
+
+Ignite UI for Angular の `IgxTextHighlight` ディレクティブは、テキストのハイライト、大文字と小文字の区別のオプション、完全一致のみのハイライトをサポートします。既にハイライトしている部分を含む、テキストのハイライトをアクティブに保持できます。
+
+### TextHighlight デモ
+<div class="sample-container loading" style="height: 260px;">
+    <iframe id="text-highlight-1-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/text-highlight-1" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="text-highlight-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">stackblitz で表示</button>
+</div>
+<div class="divider--half"></div>
+
+> [!NOTE]
+> Ignite UI for Angular コンポーネントをプロジェクトに追加する前に、必要なすべての依存関係を構成し、プロジェクトのセットアップが正しく完了したことを確認してください。[インストール](https://www.infragistics.com/products/ignite-ui-angular/getting-started#installation)のトピックで手順をご確認ください。
+
+### 使用方法
+
+Ignite UI for Angular TextHighlight ディレクティブを使用する前に **app.module.ts** ファイルに **IgxTextHighlightModule** とアプリケーションに必要な他の Ignute UI for Angular モジュールをインポートします。
+
+```typescript
+// app.module.ts
+
+...
+import { IgxButtonModule, IgxInputGroupModule,
+        IgxIconModule, IgxRippleModule, IgxTextHighlightModule } from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [..., IgxButtonModule, IgxInputGroupModule,
+                    IgxIconModule, IgxRippleModule, IgxTextHighlightModule],
+    ...
+})
+export class AppModule {}
+```
+
+次にテキストの異なる部分のハイライトに使用できる検索ボックスを作成します。Ignite UI for Angular の [InputGroup](https://www.infragistics.com/products/ignite-ui-angular/angular/components/input_group.html) コンポーネントは、一致のクリア、次の一致、前の一致へ移動するためのボタン、検索で大文字と小文字を区別を指定するボタンを追加します。また一致がいくつ見つかったかをを示すラベルがあります。
```

Comment:
You can use relative links to avoid needing to change the root of the URL for other Ignite UI for Angular topics.
Ex.
[InputGroup](input_group.html)
Otherwise, please use jp.infragistics.com.